hoomd/md/MuellerPlatheFlow.h

```diff
@@ -55,81 +33,134 @@ class PYBIND11_EXPORT MuellerPlatheFlow : public Updater
     MuellerPlatheFlow(std::shared_ptr<SystemDefinition> sysdef,
                       std::shared_ptr<ParticleGroup> group,
                       std::shared_ptr<Variant> flow_target,
-                      const flow_enum::Direction slab_direction,
-                      const flow_enum::Direction flow_direction,
+                      std::string slab_direction_str,
+                      std::string flow_direction_str,
                       const unsigned int N_slabs,
                       const unsigned int min_slab,
-                      const unsigned int max_slab);
+                      const unsigned int max_slab,
+                      Scalar flow_epsilon);
 
     //! Destructor
     virtual ~MuellerPlatheFlow(void);
 
     //! Take one timestep forward
     virtual void update(uint64_t timestep);
 
-    Scalar summed_exchanged_momentum(void) const
+    Scalar getSummedExchangedMomentum(void) const
         {
         return m_exchanged_momentum;
         }
 
-    unsigned int get_N_slabs(void) const
+    unsigned int getNSlabs(void) const
         {
         return m_N_slabs;
         }
-    unsigned int get_min_slab(void) const
+    unsigned int getMinSlab(void) const
         {
         return m_min_slab;
         }
-    unsigned int get_max_slab(void) const
+    unsigned int getMaxSlab(void) const
         {
         return m_max_slab;
         }
+    std::shared_ptr<Variant> getFlowTarget(void) const
+        {
+        return m_flow_target;
+        }
+    std::string getSlabDirection(void) const
+        {
+        return getStringFromDirection(m_slab_direction);
+        }
+    std::string getFlowDirection(void) const
+        {
+        return getStringFromDirection(m_flow_direction);
+        }
```

Comment:
I would prefer `Python` or `String` suffix these functions to indicate that they are specifically for getting the Python associated string.